.github/workflows/release-trigger.yaml

```diff
@@ -9,7 +9,9 @@ jobs:
   check-version:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Do we need all history? Or just the latest 2 commits?

Reply:
The latest 2 commits should suffice but I think it doesn't hurt to get all of the history